### ISSUE TITLE
Add Cypress accessibility tests for initial components

### ISSUE THREAD

**Issue by ad9242** [mi6/ic-ui-kit#1192]
## Summary
As part of https://github.com/mi6/ic-ui-kit/issues/1166, need to add accessibility tests for the initial 3 components selected for investigation. This work should be carried out on the duplicated repo created under https://github.com/mi6/ic-ui-kit/issues/1184

Components to test:

- [x]  IcButton
- [x]  IcTopNavigation
- [x]  IcSelect

**Comment by MI6-255** [comment]
Leave this issue until we've worked on the others, as they may come nearly for "free"

**Comment by jd239** [comment]
a11y tests will be added to existing test files such as IcSelect.cy.tsx.
Using IcSelect as an example (component testing variant), the a11y test should setup like:

- Mount component
- Check IcSelect is rendered

- A11Y TEST

- Click IcSelect to display IcMenu

- A11Y TEST

By using the approach above, a11y testing should be more comprehensive due to the increase of testing in context to the component's state.
A separate file should not be required for a11y testing

**Comment by GCHQ-Developer-299** [comment]
Finall changes added in [a PR to ui-kit-cypress](https://github.com/mi6/ui-kit-cypress/pull/8)
